lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -608,20 +637,42 @@
         return toReturn;
     }());
 
+    // mappedDefaultLocale will get the default locale and update any deprecated
+    // collation/sort order values it may use
     var mappedDefaultLocale = function () {
-        var parts = platform.builtInRegexMatch(GetDefaultLocale(), /([^_]*)_?(.+)?/);
-        var locale = parts[1];
-        var collation = parts[2];
-        var availableBcpTags = reverseLocaleAcceptingCollationValues[locale];
+        let locale = undefined;
+        let collation = undefined;
+        if (isICU) {
+            // ICU's getDefaultLocale() will return a valid BCP-47/RFC 5646 langtag
+            locale = platform.getDefaultLocale();
+            const match = platform.builtInRegexMatch(locale, /-u(-[^\-][^\-]?-[^\-]+)*(-co-[^\-]+).*/);
+            if (match) {
+                collation = callInstanceFunc(StringInstanceReplace, match[2], "-co-", "");
+                locale = callInstanceFunc(StringInstanceReplace, locale, match[2], "");
+            }
+        } else {
+            // Windows' getDefaultLocale() will return a weird RFC4646 langtag
```

Comment:
lol -- "weird" is a funny way of saying something from an outdated standard, but yeah

Reply:
nit: this is a quirk of WinGlob not Windows per se, but the program logic makes that clear